docs/release_notes/0.56.0.md

```diff
@@ -0,0 +1,16 @@
+# Release 0.56.0
+
+## Features
+- Configure parallel upgrades with UpdateConfig and MaxUnavailable (or MaxUnavailablePercentage) (#3753)
+- Bootstrap update nodegroup command (#3828)
+- Update the maxUnavailable number of nodes during parallel upgrades (#3855)
+- `Update` command can update multiple nodegroups (#3914)
+
+## Improvements
+
+- Add default tags to OIDC provider resources (#3885)
+- Do not set cgroupdriver to systemd in legacy bootstrap path (#3908)
+
+## Bug Fixes
+- Avoid cn-north-1d (#3920)
+
```

Comment:
We should get this bug fix out as well: https://github.com/weaveworks/eksctl/pull/3924

```suggestion
- Ensure GPU AMI is used when using multiple instances

```